docs/reference/cluster/nodes-usage.asciidoc

```diff
@@ -1,33 +1,62 @@
 [[cluster-nodes-usage]]
 === Nodes Feature Usage
 
-[float]
-==== Nodes usage
+Returns information on the usage of features.
 
-The cluster nodes usage API allows to retrieve information on the usage
-of features for each node.
 
-[source,js]
---------------------------------------------------
-GET _nodes/usage
-GET _nodes/nodeId1,nodeId2/usage
---------------------------------------------------
-// CONSOLE
-// TEST[setup:node]
-// TEST[s/nodeId1,nodeId2/*/]
+[[cluster-nodes-usage-api-request]]
+==== {api-request-title}
+
+`GET /_nodes/usage` +
+
+`GET /_nodes/{node_id}/usage` +
 
-The first command retrieves usage of all the nodes in the cluster. The
-second command selectively retrieves nodes usage of only `nodeId1` and
-`nodeId2`. All the nodes selective options are explained
+`GET /_nodes/usage/{metric}` +
+
+`GET /_nodes/{node_id}/usage/{metric}`
+
+
+[[cluster-nodes-usage-api-desc]]
+==== {api-description-title}
+
+The cluster nodes usage API allows you to retrieve information on the usage
+of features for each node. All the nodes selective options are explained
 <<cluster-nodes,here>>.
 
-[float]
-[[rest-usage]]
-===== REST actions usage information
 
-The `rest_actions` field in the response contains a map of the REST
-actions classname with a count of the number of times that action has
-been called on the node:
+[[cluster-nodes-usage-api-path-params]]
+==== {api-path-parms-title}
+
+`{metric}`::
+    (Optional, string) Limits the information returned to the specific metrics. 
+    A comma-separated list of the following options: 
++
+--
+    `_all`::
+        Returns all stats.
+    
+    `rest_actions`::
+        Returns the REST actions classname with a count of the number of times 
+        that action has been called on the node.
+--
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=node-id]
+
+
+[[cluster-nodes-usage-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
+
+
+[[cluster-nodes-usage-api-example]]
+==== {api-examples-title}
+
+Rest action example:
+
+`GET _nodes/usage/rest_actions`
```

Comment:
Sorry I missed this earlier. You'll want to change this example to the following. Should fix the gradle issues you're seeing.
```suggestion
[source,js]
--------------------------------------------------
GET _nodes/usage
--------------------------------------------------
// CONSOLE
// TEST[setup:node]
```